style.css

```diff
@@ -304,51 +313,79 @@ display: none;
   }
 
   .swap-class {
-    order: 4;
+    order: 1;
   }
-h1 {
- justify-content: end;
- 
-}
- 
-p {
-  width: 60%;
 
- }
+  #fs-frm {
+    max-width: 45%;
+    margin: 0 auto;
+  }
 
- .connect {
-  width: 60%;
-  color: #7f8cff;
-  margin-bottom: 0;
-  text-decoration: none;
-}
+  .header-img {
+    text-align: center;
+    border-top-left-radius: 120px;
+    padding-top: 2%;
+    background-position: center;
+  }
 
-#fs-frm {
-max-width: 45%;
-margin: 0 auto;
+  .text {
+    width: 50%;
+    padding: 2% 8%;
+    margin-left: 15%;
+  }
 
+  .header-img > h1 {
+    color: #fff;
+  }
 
-}
+  .field-spacing {
+    text-align: left;
+  }
 
-.header-img {
-  text-align: center;
-  border-top-left-radius: 120px;
-  padding-top: 2%;
-  background-position: center;
-}
+  #about {
+    background-color: #e5e5e5;
+  }
 
-.text {
-  width: 50%;
-  padding: 2% 8%;
-  margin-left: 15%;
-}
+  .about-wrapper {
+    display: flex;
+  }
 
-.header-img > h1{
-  color: #fff;
-}
+  .portfolio-about {
+    max-width: 45%;
+    margin: 0;
+    border-radius: 0;
+    padding: 142px 0 142px 142px;
+  }
 
-.field-spacing {
-  text-align: left;
-}
+  .language-layout {
+    flex-direction: row;
+    width: 51%;
+  }
+
+  .language-content {
+    flex-direction: column;
+    margin: 0 3%;
+    padding-right: 12%;
+    align-items: baseline;
+  }
+
+  .languages {
+    border-top-right-radius: 100px;
+    background-color: #fff;
+    width: 100%;
+    padding-top: 50px;
+  }
 
-}
\ No newline at end of file
+  .languages > ul > li > h1 {
+    margin-bottom: 5px;
+  }
+
+  .project-item {
+    padding-left: 30px;
+  }
+
+  .about-h1 {
+    padding-top: 0;
+    margin-top: 0;
+  }
+}
```

Comment:
- [ ] Please 🙏🏽 complete the required changes from the last code reviewer, your design must look like the Figma design:

> Kindly maintain the font size, weight, and colors on the language, skill, framework.

>  Let's add the missing drop-down image icon at the right side in the same line as the Languages

>  Let's also make some adjustments to our styling to improve the spacing of the sides of the about section to make our design close to the Figma design as much as we can below:

| Figma design | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/208250958-e7b54609-fda9-493e-a38f-75ac9ecaf1b2.png) | ![image](https://user-images.githubusercontent.com/58642949/208250968-ca675340-2d60-434a-9e6b-70afa3a68433.png) |

Reply:
no you wrong

Reply:
@franclobo